spec/factories/orchestration_template.rb

```diff
@@ -1,5 +1,17 @@
 FactoryGirl.define do
-  factory :orchestration_template do
+  # A solid class to implement require methods
+  # Cannot stub these methods because the class is used to test copying
```

Comment:
What does this mean?

Reply:
The UI is testing the logic to copy and save an orchestration template. The base `OrchstrationTemplate` is an abstract class that cannot be saved. If I stub the request methods a test template may be created. But once it is copied, the new instance does not have the request methods implemented, so the save will fail.